FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_ARP.c

```diff
@@ -639,20 +666,26 @@ ARPPacket_t *pxARPPacket;
 
 void FreeRTOS_ClearARP( void )
 {
-	memset( xARPCache, '\0', sizeof( xARPCache ) );
+	/* Function requires the second argument to be an int.
+	 *
+	 * MISRA rule 17.7 relaxed. The return value is not 
+	 * required to be used. Also improves readability    */
+	( void ) memset( xARPCache, (int)'\0', sizeof( xARPCache ) );
```

Comment:
Delete comment on lines 669-672 and replace with:

```c
 	( void ) memset( xARPCache, 0, sizeof( xARPCache ) ); 
```

Reply:
done